server/pulp/server/webservices/http.py

```diff
@@ -218,10 +162,7 @@ def uri_path():
     Return the current URI path
     @return: full current URI path
     """
-    if _is_django():
-        return unicode(_get_wsgi_environ()['REQUEST_URI'])
-    else:
-        return web.http.url(web.ctx.path)
+    return unicode(_thread_local.wsgi_environ['REQUEST_URI'])
```

Comment:
This is resolved with the inclusion of SaveEnvironWSGIHandler in the last force push.